ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -17,7 +17,9 @@ QtObject {
     property bool loggedInUserAuthenticated: false
 
     property string backButtonName: ""
-    property var currentAccount: Constants.isCppApp ? walletSectionAccounts.currentAccount: walletSectionCurrent
+    property var overview: walletSectionOverview
+    property var assets: walletSectionCurrent
+    property var currentAccount: walletSectionCurrent
```

Comment:
do we need both or we replace currentAccount with assets instead?

Reply:
Currently, I need both, as other part still use currentAccount, I haven't change everything yet, but on long run, the current account will be gone.
I kept both as changing all instances in one go, will be "heavy"